lib/elixir_hw1.ex

```diff
@@ -9,10 +9,10 @@ defmodule ElixirHW1 do
   ## Examples
 
       iex> ElixirHW1.hello()
-      :world
-
+      :ok
```

Comment:
I’m not sure if this doctest makes any sense though :)

Reply:
me neither, to be honest:)